iothub/device/src/Transport/Mqtt/MqttIotHubAdapterFactory.cs

```diff
@@ -7,11 +7,11 @@ namespace Microsoft.Azure.Devices.Client.Transport.Mqtt
 {
     internal class MqttIotHubAdapterFactory
     {
-        private readonly MqttTransportSettings settings;
+        private readonly MqttTransportSettings _settings;
```

Comment:
Good standard for underscore variable name for readonly.

Reply:
We use underscore for private members whether they are readonly or not.

If they are static, then we use s_.

If they are constant we use TitleCase.